version.go

```diff
@@ -21,4 +21,4 @@
 
 package cadence
 
-const Version = "v1.0.0-preview.40"
+const Version = "v1.0.0-preview-atree-register-inlining.39"
```

Comment:
Just a nit: probably we should keep the version on master?

Reply:
Good point! I'll manually tag and fix this once it's on master to fix it. It's currently needed to be this way due to how the go generate command which updates this works